python/nav/etc/report/report.conf.d/report.conf

```diff
@@ -640,8 +640,8 @@ $url_locationid="/report/location?locationid=$locationid";
 location {
 $description = "Short info about a location; name, number of rooms and a description.";
 $sql="
-  SELECT 
-    location.locationid, 
+  SELECT
+    location.locationid,
```

Comment:
The pre-commit hook would not let me commit this without - should I do something about it?

Reply:
I assume this is part of the commit that renames the config file. Touching the file causes the pre-commit framework to kick in, and it has a rule about stripping unnecessary trailing whitespace. As long as the change is confined to the renaming commit, it's fine, IMHO.